python/langsmith/testing/_internal.py

```diff
@@ -566,7 +566,7 @@ def sync_example(
         self,
         example_id: uuid.UUID,
         *,
-        inputs: Optional[dict] = None,
+        inputs: Optional[dict] = {},
```

Comment:
Opting to fallback to empty object in the test runner because `create_examples` currently throws if `inputs` is `None`

Reply:
gotta be careful with mutable defaults in py, should leave as None and overwrite in the function https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
O no